config.yaml

```diff
@@ -1,9 +1,9 @@
 # process names are case-insensitive
 # you can use 'master' to indicate the master channel, or a list of process names to create a group
 # you can use 'mic' to control your mic input level (uses the default recording device)
+# you can use a device's full name, i.e. "Speakers (Realtek High Definition Audio)", to bind it. this works for both output and input devices
```

Comment:
- Please distinguish between windows and linux here as well. It doesn't need to be as thorough as the explanation in `README.md`, but it should mention that for linux it uses the device's description
- The changes to this file should be replicated to the config template file under `scripts`